src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -54,16 +57,18 @@ public OverseasEntitySubmissionDtoValidator(EntityNameValidator entityNameValida
         this.trustCorporateValidator = trustCorporateValidator;
         this.historicalBeneficialOwnerValidator = historicalBeneficialOwnerValidator;
         this.updateValidator = updateValidator;
+        this.removeValidator = removeValidator;
     }
 
     public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
 
         if (isRoeUpdateEnabled && overseasEntitySubmissionDto.isForUpdate()) {
             validateFullUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
-        } else if (overseasEntitySubmissionDto.isForRemove()) {
-            // TODO Perform full validation on this Remove submission (before sending the filing details to CHIPS)
+        }  else if (overseasEntitySubmissionDto.isForRemove()) {
+            // TODO Validate when we come to implementing sending the filing details for a Remove journey to CHIPS
+            
         } else {
-            validateFullRegistrationDetails(overseasEntitySubmissionDto, errors, loggingContext);
+                validateFullRegistrationDetails(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
Formatting looks wrong in GitHub. Extra tab added?

Reply:
tab removed